### PR TITLE
Add facies demo

### PR DESCRIPTION
**Pull request by alexeyhorkin**
Add facies segmentation model and demo.

### CODE REVIEW

File: models/public/conv-deconvnet/conv-deconvnet.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# conv-deconvnet
```
Comment (dkurt) [inline models/public/conv-deconvnet/conv-deconvnet.md:1]: Maybe choose different name? Like `facies_classification`?

File: demos/python_demos/facies_classification_demo/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+```
+
+To run the demo, you can use public or pre-trained models. You can download the pre-trained models with the OpenVINO [Model Downloader](../../../tools/downloader/README.md) or from [xxx] <- (link)
+
+## Demo Output
+
+The application uses matplotlib to display resulting instance classification masks, saving the picture to the current directory.
+
+![](./facies_classification_demo.png)
```
Comment (dkurt) [inline demos/python_demos/facies_classification_demo/README.md:61]: Why not .jpg?

**Comment by dkurt** [comment]
> demos/python_demos/facies_classification_demo/libuser_cpu_extension_for_max_unpool.so

We should not put binaries into source code. Take a look at similar PR with CPU extensrions: https://github.com/openvinotoolkit/open_model_zoo/pull/949

**Comment by dkurt** [comment]
Please change target branch to `develop` (it's also better to name you branch differently from `develop`, i.e. `facies_classification`)

File: models/public/conv-deconvnet/pytorch-onnx.patch [hunk 3]
```diff
@@ -0,0 +1,157 @@
+From 1c793aede1b0f386e1a6dbff437c28fb79a605f3 Mon Sep 17 00:00:00 2001
+From: "Khorkin, Alexey" <alexey.khorkin@intel.com>
+Date: Sat, 3 Oct 2020 03:29:06 +0300
+Subject: [PATCH] conv-deconvnet-convert
```
Comment (dkurt) [inline models/public/conv-deconvnet/pytorch-onnx.patch:4]: Remove personal data from patch

File: models/public/conv-deconvnet/pytorch-onnx.patch [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+---
+ core/models/section_deconvnet.py | 45 ++++++++++++++++++----------
+ requirements.txt                 |  2 ++
+ torch2onnx.py                    | 51 ++++++++++++++++++++++++++++++++
+ 3 files changed, 82 insertions(+), 16 deletions(-)
+ create mode 100644 requirements.txt
+ create mode 100644 torch2onnx.py
```
Comment (dkurt) [inline models/public/conv-deconvnet/pytorch-onnx.patch:12]: Can patch work without this block? Seems like just a comment

File: models/public/conv-deconvnet/pytorch-onnx.patch [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+ 
+diff --git a/requirements.txt b/requirements.txt
+new file mode 100644
+index 0000000..e03d22e
+--- /dev/null
++++ b/requirements.txt
+@@ -0,0 +1,2 @@
++torch==1.5.0
++torchvision==0.6.0
+diff --git a/torch2onnx.py b/torch2onnx.py
```
Comment (dkurt) [inline models/public/conv-deconvnet/pytorch-onnx.patch:98]: Maybe just add `torch2onnx.py` file to demo folder?

Comment (alexeyhorkin) [reply models/public/conv-deconvnet/pytorch-onnx.patch:98]: i just reproduced what i saw here - https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/ctdet_coco_dlav0_384/ctdet_coco_dlav0_384.md  (as you can see, there is no convert.py file until path applying)

Comment (dkurt) [reply models/public/conv-deconvnet/pytorch-onnx.patch:87]: Anyway it's simpler to maintain as a separate script.
